arrow-docs/docs/datatypes/intro/README.md

```diff
@@ -1,170 +0,0 @@
----
```

Comment:
All gone? :(

Reply:
Not actually this one does not need to go if I can adapt it to Either but this tutorial will need to be rewritten based on some other technique that is not abstracting away MonadError since those type classes that are kinded are going away. I'll try to bring it back

Reply:
restored!